### ISSUE TITLE
show number in play in hero list instead of probability

### ISSUE THREAD

**Issue by qwervbnm** [auto-chess-ui-mod/download#8]
I think it is a very very good function that I can see how many units are in play on hero draw card panel. It is more important than just know the draw probability. For example It can change my strategy and dicision when I see the count is 3 or more than 3.

But, I can know that only when it is been showed in hero draw card panel. Could you please release an alternate version that can show number in play in hero list instead of probability.

**Comment by TiesdeKok** [comment]
I'll see what I can do. The thing is that `number in play` is not necessarily more informative than `probability to draw`. I can illustrate this using an extreme example: 

> There are currently 6 Zeus hero's in play, so there is only 4 left. You might think, "oh, I should give up trying to pull a Zeus". However, let's say that all the other legendary heroes are completely used up and in play (I know, unlikely, but for the sake of example). This would imply that there is 100% draw chance for Zeus, at least within the 5-cost units. I.e. trying to pull a Zeus would actually be quite a good strategy, even though there are already 6 of them in play. 

The best solution would probably be to let people decide for themselves what information they want, so I might indeed just add a version of the Full (without tier list) that has the number in play instead of the probability. 

Either way, thanks for the suggestion!

**Comment by qwervbnm** [comment]
Yes, you are right. These two kinds of information have different uses. 

And "number in play" version is indeed what I want. Waiting for good news. Thank you!
